lars-reimann/api-editor
# feat: `api` command should extract source code of api elements

Closes #1106.

### Summary of Changes
added an attribute `code` to functions and classes that contain the unformatted source code of the function or class


### Testing Instructions

run `package-parser api -p sklearn -o ./out` and compare the output with the original file that the element contains.
`package-parser annotations -a ../data/api/sklearn__api.json -u ../data/usages/sklearn_usages_counts.json -o ./out/annotations.json` can verify that no error occur if the file is loaded.
